app/__test__/attribute/attribute.schema.test.ts

```diff
@@ -0,0 +1,147 @@
+import { CreateAttributeInput, createAttributeSchema } from "@/app/(backend)/api/attribute/attribute.schema";
+
+it("should validate input with all required fields", () => {
+  const input = {
+    size: "small",
+    color: "#FF0000",
+    quantity: 10,
+    image: "https://example.com/image.jpg",
+  };
+
+  const result = createAttributeSchema.parse(input);
+
+  expect(result).toEqual(input);
+});
+
+it("should trim string fields", () => {
+  const input = {
+    size: "   small   ",
+    color: "   #FF0000   ",
+    quantity: 10,
+    image: "   https://example.com/image.jpg   ",
+  };
+
+  const expected = {
+    size: "small",
+    color: "#FF0000",
+    quantity: 10,
+    image: "https://example.com/image.jpg",
+  };
+
+  const result = createAttributeSchema.parse(input);
+
+  expect(result).toEqual(expected);
+});
+
+it("should throw error when size field is missing", () => {
```

Comment:
`it("should throw error when 'size' field is missing", () => {`
emphasize in size field